reddit.py

```diff
@@ -0,0 +1,44 @@
+import praw
+import requests
+import requests.auth
+import pprint
+"""
+This file takes comments from /r/totallynotrobots and generates a text file which can be the processed by another python program
+"""
+
+#PRAW setup
+infoFile = open("redditInfo.txt")
+secretInfo = infoFile.readlines()
+reddit = praw.Reddit(user_agent='text_mining', client_id=secretInfo[0].strip(), client_secret=secretInfo[1].strip(),username=secretInfo[2].strip(),password=secretInfo[3].strip())
+sub = reddit.subreddit('totallynotrobots')
```

Comment:
Instead of reading a text file, you could have just created a python file with secret keys as a variable and imported the python file.